passgen.py

```diff
@@ -0,0 +1,23 @@
+# Made the imports more readable
+import random
+import string
+
+# Removed unencessary print statements as any information should be in the projects readme file, not the actual script.
+
+# Changed text to be more concise
+def main():
+    amount = int(input('Amount of Passwords: '))
+    value = 1
+    while value <= amount:
+        code = ('').join(random.choices(string.ascii_letters + string.digits, k=24))
+        f = open('Passwords.txt', "a+")
+        # Removed f-string as its un-neccessary in this scenario. (should only be used when attempting to merge two pieces of text in a clean way.)
+        f.write(code + '\n')
+        f.close()
+        # Same thing as above, removing un-neccessary f-string
+        print(code)
+        value += 1
```

Comment:
You open, write & close the file during the loop. That's like opening a file in Notepad, writing the password in & closing it again. Then opening again, writing & closing, open, write, close, open, write, close etc.

How about we instead open, then write, write, write, and then close?
```suggestion
    f = open('Passwords.txt', "a+")
    while value <= amount:
        code = ('').join(random.choices(string.ascii_letters + string.digits, k=24))
        # Removed f-string as its un-neccessary in this scenario. (should only be used when attempting to merge two pieces of text in a clean way.)
        f.write(code + '\n')
        # Same thing as above, removing un-neccessary f-string
        print(code)
        value += 1
    f.close()
```

Reply:
hey Michael! thanks for your comment. the reason i didn't implement that is because i didn't want to completely re-write the structure for track. i'm well aware of that issue/optimization.